plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoClientConfig.java

```diff
@@ -310,15 +317,66 @@ public MongoClientConfig setImplicitRowFieldPrefix(String implicitRowFieldPrefix
         return this;
     }
 
-    public boolean getSslEnabled()
+    public boolean getTlsEnabled()
     {
-        return this.sslEnabled;
+        return this.tlsEnabled;
     }
 
-    @Config("mongodb.ssl.enabled")
-    public MongoClientConfig setSslEnabled(boolean sslEnabled)
+    @Config("mongodb.tls.enabled")
+    @LegacyConfig("mongodb.ssl.enabled")
+    public MongoClientConfig setTlsEnabled(boolean tlsEnabled)
     {
-        this.sslEnabled = sslEnabled;
+        this.tlsEnabled = tlsEnabled;
+        return this;
+    }
+
+    public Optional<@FileExists File> getKeystorePath()
```

Comment:
nit: Can we extract this as a separate config so these properties can be used if `ssl` is enabled and additionally can we extract the tls configuration in a some sort of a consumer.

Reply:
Can we have any additional validation if `tls` is disabled and these properties are configured ?

Reply:
Actually, I was going to separate a config, but I suspended it because it looks overkill in this case. 

Added a validation method to `MongoClientConfig`.